integration_tests/kafka_tests/run.sh

```diff
@@ -22,7 +22,7 @@ while ! is_kafka_ready; do
     sleep 1
 done
 
-check whether Kafka is now ready or the loop has been aborted:
+# check whether Kafka is now ready or the loop has been aborted:
```

Comment:
we don't run them often, do we? :D

Reply:
Actually I made some improvements in Avro PR -- so should be better in near future :D.